src/modules/75-cd/components/PipelineSteps/CloudFormation/CloudFormationCreateStack/InlineParameterFile.tsx

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import * as Yup from 'yup'
+// import { useParams } from 'react-router-dom'
+import { map, defaultTo } from 'lodash-es'
+import { Layout, Button, Formik, ButtonVariation, Select, FormInput, Text } from '@harness/uicore'
+import { Form, FieldArray } from 'formik'
+import { Classes, Dialog } from '@blueprintjs/core'
+// import type { ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import { useStrings } from 'framework/strings'
+// import { useCFParametersForAws } from 'services/cd-ng'
+import css from '../CloudFormation.module.scss'
+
+interface InlineParameterFileProps {
+  onClose: () => void
+  onSubmit: (values: any) => void
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
  onSubmit: (values: unknown) => void
```